src/interrex_to_movebank.Rmd

```diff
@@ -49,7 +49,12 @@ movebank_ref_data <-
   dplyr::filter(!is.na(colour_ring)) %>%
   dplyr::mutate(
     # animal-comments: NOT AVAILABLE
-    `animal-id` = metal_ring,
+    `animal-id` = 
+      if_else(
+        is.na(metal_ring),
```

Comment:
I think it reads slightly easier if you write:

`if_else(!is.na(metal_ring), metal_ring, colour_ring)`

If not A empty, pick A otherwise B